docker/php/apache/apm.ini

```diff
@@ -0,0 +1,3 @@
+elastic_apm.enabled = true
+elastic_apm.server_url = http://apm-server:8200
```

Comment:
It only will work for local deployments, not for Elastic Cloud, ECK, or k8s.

Reply:
Do we run these tests in those environments?